src/leveldb/include/leveldb/status.h

```diff
@@ -103,6 +103,8 @@ class LEVELDB_EXPORT Status {
 inline Status::Status(const Status& rhs) {
   state_ = (rhs.state_ == nullptr) ? nullptr : CopyState(rhs.state_);
 }
+
+// NOLINTBEGIN(bugprone-unhandled-self-assignment)
```

Comment:
30281: Subtree updates should not be cherry-picked, use `git subtree pull`

Reply:
Good catch!

Reply:
done